src/test/file_preload/TEST1

```diff
@@ -53,6 +53,11 @@ export PMEMFILE_POOLS=$DIR/dummy_mount_point:$DIR/dummy_fs
 export PMEMFILE_PRELOAD_LOG=pmemfile_preload.log
 export INTERCEPT_LOG=intercept.log
 
+cp repo_dummy_file_a dummy_file_a
+chmod 644 dummy_file_a
+cp repo_dummy_file_b dummy_file_b
+chmod 644 dummy_file_b
```

Comment:
This can fail when tests are executed in parallel, because test dir is shared. Please copy those files to $DIR.